rebasehelper/options.py

```diff
@@ -55,6 +55,12 @@
         "name": ["--results-dir"],
         "help": "directory where rebase-helper output will be stored",
     },
+    {
+        "name": ["--bright-color-scheme"],
```

Comment:
I would prefer `--background {dark,light,auto}`, default being `auto`, like in vim for example.